File: ArchRefClient/ArchRefClient/src/app/shared/repository.ts

import { NodeType } from './nodetype';
import { RelationshipType } from './relationshiptype';
export class Repository {

  name: string;
  id: number;

  nodeTypeList: NodeType[] = [];
  relationshipTypeList: RelationshipType[] = [];

  constructor(name: string) {
    this.name = name;
  }

  public getName() {
    return this.name;
  }

  public setName(name: string) {
    this.name = name;
  }

  public getId() {
    return this.id;
  }

  public getNodeTypeList() {
    return this.nodeTypeList;
  }

  public getRelatioshipTypeList() {
     return this.relationshipTypeList;
  }

}

File: ArchRefServer/ArchRef_Server/src/main/java/de/arthurkaul/archref/Repository.java
package de.arthurkaul.archref;

public class Repository {
	
	private String name; 
	
	public Repository(String name){
		this.setName(name);
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}
	
}
